.pre-commit-config.yaml

```diff
@@ -22,3 +22,4 @@ repos:
     hooks:
       - id: isort
         name: isort (python)
```

Comment:
```
-   repo: https://github.com/Riverside-Healthcare/djLint
    rev: v1.24.0
    hooks:
      - id: djlint-jinja
        types_or: ['html', 'jinja']
```

This was commented out on the find frontend at one point - not 100% sure why. We could add it in here and try to stick to it as it should just make sure out HTML and jinja is consistently formatted

Reply:
@Joel-Pearce do you know why the pre-commit isn't running as a task right now?

Reply:
I'm not entirely sure, I've been looking in it. From what I recall, @gidsg might have to configure something in github?

Reply:
It's working now!

Reply:
Nice!